src/webserver/api/get_calendar_week.php

```diff
@@ -1,39 +1,56 @@
 <?php
-    $format = "d.m.Y";
+    //input: http://localhost/api/get_calendar_week.php?calendarWeek=17&year=2022
+    //output: {"calendarWeek":"51","weekdays":["20.12.2021","21.12.2021","22.12.2021","23.12.2021","24.12.2021"],"year":"2021"}
+    
+    //check if file is called directly
+    if (basename(__FILE__) == basename($_SERVER["SCRIPT_FILENAME"])) {
+        //calculate offset based on calendarWeek
+        if(!isset($_GET["calendarWeek"]) || empty($_GET["calendarWeek"])) {
+            $week = date("W");
+        }else {
+            $week = htmlspecialchars($_GET["calendarWeek"]);
+        }
 
-    if(htmlspecialchars($_GET["format"]) == "en") {
-        $format = "Y-m-d";
-    }
+        if(!isset($_GET["format"]) || empty($_GET["format"])) {
+            $format = "en";
+        }else if($_GET["format"] == "de") {
+            $format = "de";
+        }
 
-    //increment or decrement $week to change calendar week
-    if(!isset($_GET["week"])) {
-        $week = -1;
-    } else {
-        $week = $_GET["week"] - date("W") - 1;
+        get_calendar_week($week, $format);
```

Comment:
Wenn du es so änderst  wie vorgeschlagen, dann musst du es so ersetzen:
```suggestion
        echo json_encode(get_calendar_week($week, $format));
```